test/WebJobs.Script.Tests/UtilityTests.cs

```diff
@@ -465,6 +465,24 @@ public void GetAccountNameFromDomain(string url, bool isUrlWithNoSas)
             Assert.Equal(isUrlWithNoSas, Utility.IsResourceAzureBlobWithoutSas(new Uri(url)));
         }
 
+        [Theory]
```

Comment:
Should we add a test case that validates that the diagnostic event has been created?

Reply:
Added an integration test for this!